### PR TITLE
Update JWT docs

### PR DESCRIPTION
**Pull request by ram-vibhul**

### CODE REVIEW

**Comment by bunin** [comment]
how did you test it?

**Comment by ram-vibhul** [comment]
> how did you test it?

just in my local env. Is it not correct?

**Comment by bunin** [comment]
> > how did you test it?
> 
> just in my local env. Is it not correct?

I don't know, that's why I'm asking how did you test it. I still don't get the answer.

Did you follow this whole doc though? Generate a keypair, generate jwt token with both golang and javastcript example, then login using these tokens and do something in ccx?

**Comment by ram-vibhul** [comment]
> > > how did you test it?
> > 
> > 
> > just in my local env. Is it not correct?
> 
> I don't know, that's why I'm asking how did you test it. I still don't get the answer.
> 
> Did you follow this whole doc though? Generate a keypair, generate jwt token with both golang and javastcript example, then login using these tokens and do something in ccx?
> 
> and can we get rid of `JWT_PUBLIC_KEY_PKIX: '1'`? it's optional, so it'll be a bit simpler if we use the other format

generated ssh key with the existing one and added in golang but it was throwing error.
so i had to change command for key generation, and then add it to golang code. Instead of k8s secret, i had to pass env variables in auth. and ran the script and jwt token generated. so logged in and account was like userID.ext-id

File: docs/admin/Customisation/JWT.md [hunk 1]
```diff
@@ -26,7 +26,6 @@ you need to set the configuration parameters in `ccx.services.auth` in ccx value
 ```
       env:
         JWT_PUBLIC_KEY_ID: 'EXAMPLE_CSP'
-        JWT_PUBLIC_KEY_PKIX: '1'
```
[the remark this answers is not in the document]
Comment (ram-vibhul) [reply docs/admin/Customisation/JWT.md]: what do you mean by does it work?

File: docs/admin/Customisation/JWT.md [hunk 2]
```diff
@@ -15,31 +15,30 @@ A **JWT** contains an associative array with claims about the user and session,
 
 The private key is used by **CSP** to encrypt the JWT token (see examples). A key pair can be generated with:
 
+ssh-keygen will produce a PKCS#1 RSA private key that starts with -----BEGIN RSA PRIVATE KEY----- and openssl to extract the public key in the desired PEM format
 ```bash
-ssh-keygen -t rsa -b 4096 -f ccx.key
+ssh-keygen -t rsa -b 4096 -m PEM -f ccx.key
+openssl rsa -in ccx.key -pubout -out ccx.key.pub
```
Comment (bunin) [inline docs/admin/Customisation/JWT.md:21]:
```suggestion
ssh-keygen -e -f ccx.key -m PEM > ccx.key.pub
```

File: docs/admin/Customisation/JWT.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bunin) [reply docs/admin/Customisation/JWT.md]: did you test it after this change?

**Review by bunin — APPROVED** [review]

**Pull request merged**